memo/memo_workflow.go

```diff
@@ -0,0 +1,144 @@
+package memo
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"strings"
+	"time"
+
+	"go.temporal.io/api/common/v1"
+	workflowpb "go.temporal.io/api/workflow/v1"
+	"go.temporal.io/sdk/activity"
+	"go.temporal.io/sdk/client"
+	"go.temporal.io/sdk/converter"
+	"go.temporal.io/sdk/log"
+	"go.temporal.io/sdk/workflow"
+)
+
+/*
+ * This sample shows how to use memo. (Note this feature only work with Elasticsearch)
```

Comment:
Why does it only work with Elasticsearch?

Reply:
It was copy-paste, just updated PR to fix those comments.